### PR TITLE
Health check

### LINKED ISSUE #707: [v1] HealthCheck endpoint

### PR DESCRIPTION
**Pull request by sekulicd**
This adds health check endpoint.

This closes #707 

@altafan please review.

### CODE REVIEW

**Comment by altafan** [comment]
@sekulicd let's add the healthcheck handler to the operator service too.

Also, we should update permissions_test.go to make it import health proto and test that all its methods are in the whitelist as done for other protos.

**Comment by sekulicd** [comment]
> @sekulicd let's add the healthcheck handler to the operator service too.
> 
> Also, we should update permissions_test.go to make it import health proto and test that all its methods are in the whitelist as done for other protos.

Done

File: internal/interfaces/grpc/permissions/permissions_test.go [hunk 1]
```diff
@@ -34,6 +36,10 @@ func TestWhitelistedMethods(t *testing.T) {
 		allMethods = append(allMethods, fmt.Sprintf("/%s/%s", tdexv2.TradeService_ServiceDesc.ServiceName, v.MethodName))
 	}
 
+	for range grpchealth.Health_ServiceDesc.ServiceName {
+		allMethods = append(allMethods, fmt.Sprintf("/%s/%s", grpchealth.Health_ServiceDesc.ServiceName, "Check"))
+	}
```
Comment (altafan) [inline internal/interfaces/grpc/permissions/permissions_test.go]:
```suggestion
	for _, v := range grpchealth.Health_ServiceDesc.ServiceName {
		allMethods = append(allMethods, fmt.Sprintf("/%s/%s", grpchealth.Health_ServiceDesc.ServiceName, v.MethodName))
	}
```

This implies adding Watch to the whitelist even if it's not implemented basically

Comment (sekulicd) [reply internal/interfaces/grpc/permissions/permissions_test.go]: Is it better if we dont add it to whitelist cause it is not implemented?

Comment (altafan) [reply internal/interfaces/grpc/permissions/permissions_test.go]: alright, so this should change like: 
```suggestion
	allMethods = append(allMethods, fmt.Sprintf("/%s/%s", grpchealth.Health_ServiceDesc.ServiceName, "Check"))
```

**Review by altafan — APPROVED** [review]

**Pull request merged**